packages/pilot/src/containers/TransactionDetails/style.css

```
@@ -17,3 +18,242 @@
+    }
+  }
+
+  .paymentInfo {
+    align-items: baseline;
+    flex-wrap: wrap;
+    border-bottom: 1px solid var(--color-light-iron-50);
+    justify-content: space-around;
+
+    & > div:first-of-type {
+      width: 100%;
+    }
+  }
+
+  .detailsInfo {
+    flex-direction: column-reverse;
+    justify-content: space-between;
+
+    & h2 {
+      margin: 16px 0;
+      color: var(--color-black);
+      font-size: 16px;
+      text-transform: inherit;
+    }
+  }
+
+  .recipientsInfo {
+    align-items: flex-start;
+    display: flex;
+    max-height: 1300px;
+    min-height: 1100px;
+    padding: 0;
+  }
+
+  .customerInfo {
+    border-bottom: 1px solid var(--color-light-iron-50);
+    color: var(--color-black);
+    order: 1;
+
+    & > div > div > div > div > div {
+      flex-direction: row;
+
+      & > div {
+        padding-bottom: 8px;
+        padding-right: 16px !important; /* stylelint-disable-line declaration-no-important */
+
+        & > div {
+          display: flex;
+          justify-content: space-between;
+        }
+
+        & h4 {
+          font-size: 16px;
+          font-weight: bold;
+
+          &:after {
+            content: ": ";
+            padding-right: 3px;
+          }
+        }
+
+        & span {
+          font-size: 14px;
+        }
+      }
+    }
+  }
+
+  .transactionCardInfo {
+    color: var(--color-black);
+    min-height: 450px;
+
+    & > div {
+      padding-top: 16px;
+    }
+
+    & h2 {
+      color: var(--color-black);
+      font-size: 16px;
+    }
+
+    & > div > div > div > div > div {
+      flex-direction: row;
+
+      & > div {
+        padding-bottom: 8px;
+        padding-right: 16px !important; /* stylelint-disable-line declaration-no-important */
+
+        & > div {
+          display: flex;
+          justify-content: space-between;
+        }
+
+        & h4 {
+          font-size: 16px;
+          font-weight: bold;
+
+          &:after {
+            content: ": ";
+            padding-right: 3px;
+          }
+        }
+
+        & span {
+          font-size: 14px;
+        }
+      }
+    }
+  }
+
+  .transactionInfo {
+
+    & > div {
+      margin: 0;
+      padding: 0;
+      width: 100%;
+    }
+  }
+
+  .eventsList {
+    width: 100%;
+
+    & h2 {
+      color: var(--color-black);
+      font-size: 16px;
+      text-transform: inherit;
+    }
+  }
+  /* stylelint-disable */
+  aside + div > header,
+  aside div,
+  aside nav,
+  aside button {
+    display: none !important;
+  }
+
+  aside {
+    background: var(--color-white);
+    display: flex;
+    min-height: 60px !important;
+    text-align: center;
+    width: 100% !important;
+
+    & > header {
+      justify-content: center !important;
+    } 
+  }
+
+  body {
+
+    & > div > div {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+    }
+  }
+  /* stylelint-enable */
+
+  html   {
```

tem espaços a mais depois de `html`. se esse estilo fosse super importante, desabilitaria o stylelint também - embora eu acha melhor selecionar `body` invés de `html` pro stylelint parar de reclamar mesmo.

```suggestion
  html {
```